Formula/b/blisp.rb

```diff
@@ -0,0 +1,26 @@
+class Blisp < Formula
+  desc "ISP tool & library for Bouffalo Labs RISC-V Microcontrollers and SoCs "
+  homepage "https://github.com/pine64/blisp"
+  license "MIT"
+  url "https://github.com/pine64/blisp.git",
+      tag:      "v0.0.4",
+      revision: "f601b6b965a922b4a9f400d1c8fd946b69708728"
+  head "https://github.com/pine64/blisp.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    # Remove unrecognized options if they cause configure to fail
+    # https://rubydoc.brew.sh/Formula.html#std_configure_args-instance_method
+    mkdir "build" do
+        system "cmake", "..", "-DBLISP_BUILD_CLI=ON"
+        system "cmake", "--build", "."
+      end
+    bin.install "build/tools/blisp"
+  end
+
+  test do
+  # Verify that the binary can be executed and output is correct
+    assert_match "blisp", shell_output("#{bin}/blisp --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Sadly, I don't really know what to test else. This software is relatively basic...

Reply:
Does `blisp` have any other commands other than `--version`?

Reply:
It does. But it requires to have a RISC-V machine connected to the host machine

Reply:
Can we run them anyway to check for the expected error when an RISC-V machine is not connected?

See also the advice in the link I mentioned above:

> If the software cannot function without credentials or requires a virtual machine, docker instance, etc. to run, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected. This is preferred over mocking a dependency.

Finally, it's probably good to avoid marking reviews as resolved unless the reviewer thinks they're resolved. You keep marking this review as resolved when it very clearly hasn't been.

Reply:
Did just that